src/Sulu/Bundle/ContentBundle/Resources/views/Template/macros/add_button.html.twig

```diff
@@ -2,5 +2,7 @@
     <div id="mapper-add-{{ property.name }}" class="addButton" data-mapper-add="{{ property.name }}">
         <span class="control-label">Add {{ property.name }}</span>
     </div>
-    <span class="desc">(<span id="current-counter-{{ property.name }}">{{ property.minOccurs }}</span> out of <span>{{ property.maxOccurs }}</span>)</span>
+    {% if property.maxOccurs is not null %}
```

Comment:
I just do not display the counter if the `maxOccurs` property is not set. The alternative would be to display something like `2 out of ∞`